pkg/service/main.go

```diff
@@ -59,17 +60,18 @@ func Main(serviceName string, serviceVersion string, proto interface{}, ctx cont
 		startupTimer,
 		dic,
 		[]interfaces.BootstrapHandler{
+			handlers.SecureProviderBootstrapHandler,
 			httpServer.BootstrapHandler,
 			clients.NewClients().BootstrapHandler,
 			NewBootstrap(router).BootstrapHandler,
 			autodiscovery.BootstrapHandler,
-			message.NewBootstrap(serviceName, serviceVersion).BootstrapHandler,
+			handlers.NewStartMessage(serviceName, serviceVersion).BootstrapHandler,
 		})
 
 	ds.Stop(false)
 }
 
-func setServiceName(name string, profile string) string {
+func setServiceName(name string, _ string) string {
```

Comment:
Since this is an internal function and this parameter clearly isn't being used anymore, wouldn't it be better to just remove it altogether vs. using a blank identifier?

Also as this is seemingly unrelated to the SecretProvider changes, IMHOP it belongs in its own commit.

Reply:
I was just fixing minor issues called out by the IDE. I agree refactoring it to remove is probably a good idea for another PR.